### PR TITLE
Updated Cancel on payment modal

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#5855

*Description of changes:*
- fixed Confirm Name Request dialog title
- moved Cancel button to left
- added Close button

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/src/components/payment/payment-modal.vue [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <template>
-  <v-dialog max-width="40%" :value="isVisible" persistent>
-    <v-card class="pa-9">
-      <v-card-text class="h3">
-        Confirm Name Request
+  <v-dialog max-width="50%" :value="isVisible" persistent>
+    <v-card>
+      <v-card-title class="d-flex justify-space-between">
+        <div>Confirm Name Request</div>
```
Comment (cameron-freshworks) [inline client/src/components/payment/payment-modal.vue:5]: Just curious why a Div over a span here? Feels more like an inline element opposed to a block/section.. though i could be wrong!

**Review by cameron-freshworks — APPROVED** [review]
Looks good!

File: client/src/components/payment/payment-modal.vue (hunk 1 above)
Comment (severinbeauvais) [reply client/src/components/payment/payment-modal.vue:5]: Span works as well. Flexbox doesn't seem to care.

Comment (cameron-freshworks) [reply client/src/components/payment/payment-modal.vue:5]: Cool!

**Review by vysakh-menon-aot — APPROVED** [review]

**Pull request merged**